src/Analyzer.Utilities/Analyzer.Utilities.csproj

```diff
@@ -1,78 +1,30 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
   <ImportGroup Label="Settings">
     <Import Project="..\..\build\Targets\Analyzers.Settings.targets" />
   </ImportGroup>
   <PropertyGroup>
-    <MinimumVisualStudioVersion>12.0</MinimumVisualStudioVersion>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{F8502E24-5EB1-4CDC-8887-1A0CC8008D96}</ProjectGuid>
-    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <PackageTargetFallback>portable-net45+win8+wp8+wpa81</PackageTargetFallback>
+    <AnalyzerProject>true</AnalyzerProject>
     <RootNamespace>Analyzer.Utilities</RootNamespace>
     <AssemblyName>Analyzer.Utilities</AssemblyName>
```

Comment:
This will be implicit right? (Since the project file has the same name). You can run [https://github.com/srivatsn/MSBuildSdkDiffer](https://github.com/srivatsn/MSBuildSdkDiffer) to get a report of what properties\items in the project will be implicit in the SDK.